.pre-commit-hooks.yaml

```diff
@@ -13,18 +13,22 @@
   entry: demisto-sdk validate
   pass_filenames: true
   language: python
+  require_serial: true
   files: *files
 
 - id: format
   name: format
   description: Format non-code files, to comply with XSOAR standards.
   entry: demisto-sdk format
+  args: ["-ud", "-y", "--no-validate"]
```

Comment:
why `--no-validate`?

Reply:
we have a validate hook (which is enabled by default), I don't want to run `validate` twice.
`demitso-sdk --format` will format the file and run the validate hooks after